user_tools/README.md

```diff
@@ -43,8 +43,10 @@ set python environment to version [3.8, 3.10]
       ```
 3. Make sure to install CSP SDK if you plan to run the tool wrapper.
 
-## Usage and supported platforms
+## Usage and Supported Platforms
 
 Please refer to [spark-rapids-user-tools guide](https://github.com/NVIDIA/spark-rapids-tools/blob/main/user_tools/docs/index.md) for details on how to use the tools
 and the platform.
 
+## What's New
```

Comment:
Nit: Although it is correct, better style to have an empty line following section header that separates it from the paragraph text.

Reply:
Added new line